storybookjs/storybook
# CLI: Add init support for qwik projects

## What I did
I added a generator for the qwik framework. It points to my library outside of the storybook mono repo `storybook-framework-qwik`.

## How to test

- [X] Does this need a new example in the kitchen sink apps? - repro template added (with some catches)
- [x] Does this need an update to the documentation? - I will plan 
If your answer is yes to any of these, please make sure to include it in your PR.


Open questions:
* I would mostly like confirmation that my general approach isn't totally off-base.
    * I'm assuming it's preferred to keep the Qwik framework outside the storybook monorepo until it is more stable. Is this right?
    * Is it okay that I made changes to the `baseGenerator` to support the external framework?
    * Is it okay that my framework includes a renderer built in? Qwik is only used with Vite. Because of this, I use framework and renderer interchangeably in a few places.
* ~The Qwik SB framework currently fails if there are any MDX stories set. I need to find a way to remove the mdx stories from the generated main.js file~ fixed
* ~Generating the repro template fails to launch storybook when it tries "storybook not found", but running `yarn storybook` manually afterwards works. I don't know why this is, and need to look into it more.~
